### ISSUE TITLE
Support building with OpenSSL 3.0

### ISSUE THREAD

**Issue by moschlar** [cisco/cjose#120]
There is #114 but that seems to break building with OpenSSL 1.1.1 so that's probably not the best solution.

@linuxwolf Is there an official position regarding this?

**Comment by zandbelt** [comment]
see also #103

**Comment by zandbelt** [comment]
support was added here:
https://github.com/zmartzone/cjose/commit/c345a1ae4d41856ee5aa7a93f1cbad838649632c
and is now part of the maintenance fork here:
https://github.com/OpenIDC/cjose/tree/version-0.6.2.x

**Comment by moschlar** [comment]
Huge thanks @zandbelt - I'll wait for you to finish catching up with outstanding issues/pull requests and then you will probably cut an actual release 0.6.2, right?

**Comment by zandbelt** [comment]
yes, I'll let you know when that is done (I'm pretty close but I'll give people here some time to kick in)

**Comment by TheStormN** [comment]
Fixed by https://github.com/cisco/cjose/commit/4cdd7e2f45ffd9cc9539ec01d796f1ac8c017015
